android-studio: Add at 2024.3.2.14

sammypanda opened the pull request:
# Doesn't install
Error: install: transaction: No such name: interpreter(/system/bin/linker(386))

ermo commented:
Converting to draft because of the "Doesn't install" header in the PR cover letter.

sammypanda commented:
> Converting to draft because of the "Doesn't install" header in the PR cover letter.

do you know how I can fix this kinda error?

sammypanda commented:
> Hey @sammypanda,
> 
> Can you please run boulder with this recipe and provide the output in a paste (such as https://bpa.st/) as a comment here.
> 
> This will allow others to quickly see what actually failed and where to then try and diagnose.
> 
> Can you also confirm you're using the rebuild repo rather than the old main repo for building?

I did a boulder build, i think that's the right thing? https://pastebin.com/EqJK8xVv -- it says successful though at this stage. The error comes when ``sudo moss install android-studio``

yes, sorry, should've put it from the start

yes, using the rebuild one (this is the location i have set for it https://infratest.aerynos.dev/vessel/volatile/x86_64/stone.index)

sammypanda commented:
As of matrix chat @NomadicCore 

Yah i must've done the mv-local correct from the start, it shows the same issue as the original one. Tbf it's probably something very simple inside the boulder, but i just don't have the context to know what it is. 

Like there is probably *something* that needs to be imported in, i tried looking for matches to what it's looking for and couldn't find anthing. I don't know (with the knowledge i have) how i'd be able to figure out what is needed or if AerynOS has it yet. 

![image](https://github.com/user-attachments/assets/cdf55c18-248e-44ec-8230-b9718167b7da)

Important content of image:
```
❯ just mv-local
renamed 'android-studio-2024.3.2.14-1-1-x86_64.stone' -> '/home/sammy/.cache/local_repo/x86_64/android-studio-2024.3.2.14-1-1-x86_64.stone'
renamed 'android-studio-32bit-dbginfo-2024.3.2.14-1-1-x86_64.stone' -> '/home/sammy/.cache/local_repo/x86_64/android-studio-32bit-dbginfo-2024.3.2.14-1-1-x86_64.stone'
renamed 'android-studio-dbginfo-2024.3.2.14-1-1-x86_64.stone' -> '/home/sammy/.cache/local_repo/x86_64/android-studio-dbginfo-2024.3.2.14-1-1-x86_64.stone'
Indexing 3 files

Indexed android-studio-32bit-dbginfo-2024.3.2.14-1-1-x86_64.stone
Indexed android-studio-dbginfo-2024.3.2.14-1-1-x86_64.stone
Indexed android-studio-2024.3.2.14-1-1-x86_64.stone

Index file written to "/home/sammy/.cache/local_repo/x86_64/stone.index"

░▒▓ 󰌽 sammy@vanilla   …/recipes/a/android-studio   android-studio !   16:33  
❯ sudo moss sync -u
[sudo: authenticate] Password: 
Refreshed local
Refreshed unstable
No packages to sync

░▒▓ 󰌽 sammy@vanilla   …/recipes/a/android-studio   android-studio !   16:33  
❯ sudo moss install android-studio
Error: install: transaction: No such name: interpreter(/system/bin/linker(386))
```

sammypanda commented:
okie so it doesn't install because boulder when scanning for dependencies finds one related to android

interpreter(/system/bin/linker(386))

would look more like /lib/ld for Linux :3

I don't know how to exclude parts from the scan 🫡
